pallets/msa/src/replay_tests.rs

```diff
@@ -198,6 +210,11 @@ pub fn add_signature_replay_fails() {
 				Msa::register_signature(sig1, tc.mortality),
 				Error::<Test>::SignatureAlreadySubmitted,
 			);
+			run_to_block(tc.run_to + 1);
```

Comment:
Should this be just:
```suggestion
			run_to_block(tc.mortality);
```

Reply:
Yeah I think that's better.